integration/src/main/resources/configs/integration/ReferencedBigQueryResourceLifecycle.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "ReferencedBigQueryResourceLifecycle",
+  "description": "CRUD BQ dataset and table operations",
+  "serverSpecificationFile": "workspace-local.json",
+  "kubernetes": {},
+  "application": {},
+  "testScripts": [
+    {
+      "name": "ReferencedBigQueryResourceLifecycle",
+      "numberOfUserJourneyThreadsToRun": 1,
+      "userJourneyThreadPoolSize": 1,
+      "expectedTimeForEach": 5,
+      "expectedTimeForEachUnit": "MINUTES",
+      "parameters": {
+        "spend-profile-id": "wm-default-spend-profile",
+        "bq-dataset": "projects/terra-kernel-k8s/datasets/terra_wsm_test_dataset",
+        "bq-table": "projects/terra-kernel-k8s/datasets/terra_wsm_test_dataset/tables/terra wsm test data table",
```

Comment:
I'm confused how the spaces work in the table name. Like this doesn't look like a valid FQID.

Reply:
It looks a bit weird, but spaces are explicitly allowed in BQ table names: https://cloud.google.com/bigquery/docs/tables#table_naming